### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by sbellem**

### CODE REVIEW

File: cryptoconditions/type_registry.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 
 class TypeRegistry:
+    """ """
```
Comment (sohkai) [inline cryptoconditions/type_registry.py]: Don't really see a point in this...

Comment (sbellem) [inline cryptoconditions/type_registry.py]: look closer :smirk:

**Comment by sohkai** [comment]
Mmm, not sure if we should bother putting this up on readthedocs. That also seems like a different PR, and if done, should probably take the entire README with it.

File: cryptoconditions/type_registry.py (hunk 1 above)
Comment (sbellem) [inline cryptoconditions/type_registry.py]: hint: https://github.com/bigchaindb/org/wiki/Documentation

**Comment by sbellem** [comment]
> Mmm, not sure if we should bother putting this up on readthedocs. 

What do you mean by "this", the docs?

**Comment by sbellem** [comment]
> That also seems like a different PR, and if done, should probably take the entire README with it.

Different PR from what? I am confused.

**Comment by sohkai** [comment]
Well, the sphinx generated docs. Those should be separated from the docstrings since it's a fairly large change.

**Comment by ttmc** [comment]
Suggesting that the PR get broken into two pieces is just a make-work project at this point. Let's just merge this PR and we can make the docs better/all-together in future pull requests.

**Pull request merged**